common/journal_entries/dvg_lotharingia.txt

```diff
@@ -0,0 +1,337 @@
+je_lotharian_house_of_cards = {
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+
+	is_shown_when_inactive = {
+		exists = c:LOT
+		c:LOT = ROOT
+		NOT = { has_variable = house_of_cards_complete }
+		has_law = law_type:law_autocracy
+	}
+
+	possible = {
+		ig:ig_armed_forces = {
+			is_in_government = yes
+		}
+	}
+
+	immediate = {
+		set_variable = {
+			name = aigle_var
+			value = 0
+		}
+	}
+	on_monthly_pulse = {
+		effect = {
+			if = {
+				limit = {
+					ig:ig_armed_forces = {
+						is_powerful = yes
+						is_in_government = yes
+					}
+					
+					
+					
+				}
+				change_variable = {
+					name = aigle_var
+					add = 1
+				}
+			}
+		}
+	}
+
+	complete = {
+		# If-else clause to avoid errors because of missing variable before JE activates
+		trigger_if = {
+			limit = { exists = var:aigle_var }
+			scope:journal_entry = { is_goal_complete = yes }
+		}
+		trigger_else = {
+			custom_tooltip = je_lotharian_collapse

```

Comment:
Error: custom_tooltip trigger [ Unknown loc key je_lotharian_collapse ]
  Script location: file: common/journal_entries/dvg_lotharingia.txt line: 50

Reply:
removed this